### PR TITLE
South Africa - Chandre De Wet - Github - 1

### PR DESCRIPTION
**Pull request by chandredewet**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

**Comment by DouglasVDM** [comment]
Hi Chandre, You are making good progress. No judgment for not completing the homework. You are very busy setting up our coursework.

File: homework.md [hunk 1]
```diff
@@ -4,25 +4,25 @@
...
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+Light
```
Comment (Samuelkau) [inline homework.md:13]: Opposite of black could be light, or maybe white?

File: homework.md [hunk 2]
```diff
@@ -4,25 +4,25 @@
...
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+Don't have a calculator
```
Comment (Samuelkau) [inline homework.md:19]: (",) I will help you here answer is 12240

File: homework.md [hunk 3]
```diff
@@ -4,25 +4,25 @@
...
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+(Too tired to complete this)
```
Comment (Samuelkau) [inline homework.md:25]: I will have to bring you an energy drink lol. The answer for this is Hypertext Markup Language

**Review by Samuelkau — CHANGES_REQUESTED** [review]

File: homework.md (hunk 3 above)
Comment (Samuelkau) [reply homework.md:25]: Try and complete the other questions and I will assist/review them further - happy research (",)

**Pull request closed without merging**